File: examples/mploed-applicant-scoring/src/test/java/io/zenwave360/example/applicantscoring/repository/jpa/BaseRepositoryIntegrationTest.java
package io.zenwave360.example.applicantscoring.repository.jpa;

import io.zenwave360.example.applicantscoring.config.DockerComposeInitializer;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.ActiveProfiles;

@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK)
@ActiveProfiles("test")
@DockerComposeInitializer.EnableDockerCompose
@org.springframework.transaction.annotation.Transactional
public abstract class BaseRepositoryIntegrationTest {
}

File: examples/order-fulfillment-kotlin/src/test/kotlin/io/zenwave360/example/orderfulfillment/BaseRepositoryIntegrationTest.kt
package io.zenwave360.example.orderfulfillment

import io.zenwave360.example.orderfulfillment.config.DockerComposeInitializer
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.test.context.ActiveProfiles

@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK)
@ActiveProfiles("test")
@DockerComposeInitializer.EnableDockerCompose
@org.springframework.transaction.annotation.Transactional
abstract class BaseRepositoryIntegrationTest
